### PR TITLE
Implement Key encryption decryption

### PR DESCRIPTION
**Pull request by NikithaMedikonda**
# Implement Key encryption decryption

- This pull request introduces functions for encryption and decryption of a private key.
- Also, handles testing for all the edge cases using Jest.

### CODE REVIEW

**Review by anjanibarlapati — COMMENTED** [review]
- Test step in workflow is not generating coverage.
- Multiple console warnings were there in test step. Fix them.
-  There are few commits with same commit messages. Try to write commit messages as per the changes you make and make sure they are different.

File: src/services/keyGeneration.ts [hunk 1]
```diff
@@ -9,8 +9,7 @@ export const KeyGeneration = async () => {
     const publicKey = keys.public;
     const privateKey = keys.private;
     return { publicKey, privateKey };
-  } catch (error) {
-    
-    return null;
+  } catch (error: any) {
+    throw new Error(error.message || 'Generation error');
```
Comment (anjanibarlapati) [inline src/services/keyGeneration.ts:13]: Write error message completely `Generation error`. Don't use any type for error.

File: jest.setup.js
Comment (anjanibarlapati) [inline jest.setup.js:1]: Though this is a temporary solution fix the issue. It is better to avoid mocking to fix console warns and errors. 
And don't disable ESLint in any file.

Comment (NikithaMedikonda) [reply jest.setup.js:1]: We are planning to work on these this later. We have few dependencies on this.

**Review by SoumyaKatta — APPROVED** [review]
LGTM.
